packages/ozone/tests/content-tagger.test.ts

```diff
@@ -0,0 +1,125 @@
+      dbPostgresSchema: 'ozone_content_tagger_test',
+    })
+    sc = network.getSeedClient()
+    modClient = network.ozone.getModClient()
+    await basicSeed(sc)
+    await network.processAll()
+  })
+
+  afterAll(async () => {
+    await network.close()
+  })
+
+  const getStatus = async (subject: string) => {
+    const { subjectStatuses } = await modClient.queryStatuses({
+      subject,
+    })
+
+    return subjectStatuses[0]
+  }
+
+  describe('lang tagger', () => {
+    it('Adds language tag to post from text', async () => {
+      const createPostAndReport = async (text: string) => {
+        const post = await sc.post(sc.dids.carol, text)
+        await network.processAll()
+        const report = await sc.createReport({
+          reasonType: REASONSPAM,
+          subject: {
+            $type: 'com.atproto.repo.strongRef',
+            uri: post.ref.uriStr,
+            cid: post.ref.cidStr,
+          },
+          reportedBy: sc.dids.alice,
+        })
+
+        return { post, report }
+      }
+      const [japanesePost, greekPost] = await Promise.all([
+        createPostAndReport('Xで有名な人達＋反AIや絵描きによくない'),
+        createPostAndReport(
+          'Λορεμ ιπσθμ δολορ σιτ αμετ, μει θτ vιδιτ νοστρθμ προπριαε',
+        ),
+      ])
+
+      const [japanesePostStatus, greekPostStatus] = await Promise.all([
+        getStatus(japanesePost.post.ref.uriStr),
+        getStatus(greekPost.post.ref.uriStr),
+      ])
+
+      expect(japanesePostStatus.tags).toContain('lang:ja')
+      expect(greekPostStatus.tags).toContain('lang:el')
+    })
+
+    it('Uses name/description text for language tag for list', async () => {
+      const createListAndReport = async (
+        name: string,
+        description?: string,
+      ) => {
+        const list = await sc.createList(sc.dids.carol, name, 'mod', {
+          description,
+        })
+        await network.processAll()
+        const report = await sc.createReport({
+          reasonType: REASONSPAM,
+          subject: {
+            $type: 'com.atproto.repo.strongRef',
+            uri: list.uriStr,
+            cid: list.cidStr,
+          },
+          reportedBy: sc.dids.alice,
+        })
+        return { list, report }
+      }
+
+      const [listWithDescription, listWithoutDescription] = await Promise.all([
+        createListAndReport(
+          'よくない',
+          'Xで有名な人達＋反AIや絵描きによくない感情を持つ人達＋絵描き詐称',
+        ),
+        createListAndReport('人達＋反AIや絵描きによくない感情'),
+      ])
+
+      const [japaneseListStatus, chineseListStatus] = await Promise.all([
+        getStatus(listWithDescription.list.uriStr),
+        getStatus(listWithoutDescription.list.uriStr),
+      ])
+
+      expect(japaneseListStatus.tags).toContain('lang:ja')
+      expect(chineseListStatus.tags).toContain('lang:ja')
+    })
+  })
+
+  describe('embed tagger', () => {
+    it('Adds image tag to post with image', async () => {
+      const post = sc.posts[sc.dids.carol][0]
```

Comment:
This is a record-with-media embed.  if we add a test for `sc.replies[sc.dids.bob][0]` I think we'll hit the direct image embed case.